conductor_api/src/interface.rs

```diff
@@ -93,9 +109,61 @@ impl ConductorApiBuilder {
     /// Finish the building and retrieve the populated handler
     pub fn spawn(mut self) -> IoHandler {
         self.setup_info_api();
+        self.setup_call_api();
         *self.io
     }
 
+    /// Adds a "call" method for making zome function calls
+    fn setup_call_api(&mut self) {
+        let instances = self.instances.clone();
+        let instances_map = self.instances_map.clone();
+        self.io.add_method("call", move |params| {
+            let params_map = Self::unwrap_params_map(params)?;
+            let public_id_str = Self::get_as_string("id", &params_map)?;
```

Comment:
A little bit unsure about using `"id"` to identify the instance. Only because it might be confused with the JSON-RPC field also called `"id"` which is a unique identifier for each call.

Maybe call it `"instanceId"` instead but this might also be misleading

Reply:
interesting. all the admin routes use "id" except the one that have to distinguish between interface_id and instance_id.  So I think for now I'll leave it, until will decide to make all of those also be "instance_id".

Reply:
I'm also a little unsure about this. @zippy the difference here is that the admin routes are all REST-like, e.g. `admin/instance/add` where it's clear what thing you're dealing with. `call` doesn't suggest a particular class of things that the ID would be associated with. That's why `id` feels a little ambiguous here.

Reply:
Ok.  That makes sense.